xivo_dao/alchemy/tests/test_pjsip_transport.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from hamcrest import assert_that, has_properties, contains, contains_inanyorder
+from xivo_dao.tests.test_dao import DAOTestCase
+from ..pjsip_transport import PJSIPTransport
+
+
+class TestPJSIPTransport(DAOTestCase):
```

Comment:
I would like to see some specific test about options (ex: if option duplicated, if we remove option, if we add when there are already options) and check the state of the database (PJSIPTransportOption table if option are correctly deleted)
Also tests on eq/ne (I guess they are used in confd?)
Persistor tests are too generic and implicit for these use cases

Reply:
I have not been able to find a test for `__ne__` that would fail without the method. I implemented it to be symmetrical with the `__eq__`